catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -72,9 +72,19 @@ define([
     function translateFilterToBasicMap(filter){
         var propertyValueMap = {};
         var downConversion = false;
-        if (filter.filters){
+
+        if (filter.filters !== null && isAnyDate(filter)) {
```

Comment:
✏️ Could this `if (isAnyDate(filter))` be added in the body of `if (filter.filters !== null)` on line 85 instead?